src/Chapter1.hs

```diff
@@ -614,8 +628,12 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
+sumLast2 :: Int -> Int
+sumLast2 n =
+  let units = mod n 10
+      dozen = mod (div (n - units) 10) 10
+  in units + dozen
```

Comment:
Nice solution! Though, you can notice that you can reuse the `lastDigit` function, you've already implemented, instead of `mod x 10`. This is not critical, just a suggestion :slightly_smiling_face: 
Alternatively, you can have a look into the `divMod` function, that returns the result of `div` and `mod` at the same time.

And one more suggestion: I think you don't need to do `n - units`, and `div n 10` will work jus fine. Maybe I'm missing some corner case with negative numbers, and you thought about this more than I :slightly_smiling_face: